### ISSUE TITLE
Insert <br> for linebreaks.

### ISSUE THREAD

**Issue by TheDemystifier** [typora/typora-issues#5417]
- [ YES] Searched existing issues to avoid creating duplicates.

- [YES] Confirmed that it can be reproduced in built-in themes without customized css.

   If it only exists in 3rd party themes or css, you can still report it, but please attach the theme target or the css file. We may not "fix" it, if it is caused by 3rd party themes or css styles that we cannot support.

- [YES] Searched http://support.typora.io/

----

**Describe the bug**
Linebreaks are not carried out when pasting text from Typora elsewhere.

**To Reproduce**
1. Go to 'Text Editor'
2. Write two lines with a (SHIFT+ENTER) linebreak. Example:
ABCD
EFGH
3. Copy the two lines as HTML code.
4. Paste the text somewhere. It shows like this:
<p>ABCD
EFGH</p>

**Expected behavior**
The linebreak should be preserved in the HTML code for linebreaks.
<p>ABCD<br>
EFGH</p>
Basically, the SHIFT+ENTER linebreak is completely ignored.

**Desktop (please complete the following information):**
 - OS: [Windows 10 64bit]

**Typora Version**
e.g: 1.2.5

**Comment by TheDemystifier** [comment]
Found a solution. Albeit a bit of a nuisance.
You just need to append a double space after every line that has a linebreak. Otherwise, Markdown will ignore the linebreak.
